recipe/meta.yaml

```
@@ -1,14 +1,15 @@
 {% set name = "bump-my-version" %}
-{% set version = "0.29.0" %}
-{% set build_number = 1 %}
+{% set version = "0.30.0" %}
+{% set build_number = 0 %}
+{% set python_min = "3.9" %}
```

```suggestion
{% set python_min = "3.10" %}
```

To support the use of `|` type union operator!

---

What is baffling me is that my Python 3.9 tests passed.

---

Yeah very weird, it feels like they shouldn't have! This file is missing `from __future__ import annotations`: https://github.com/callowayproject/bump-my-version/blob/master/bumpversion/utils.py

---

I changed the is_subpath function to use Union in a PR I have open.